docs/index.md

```diff
@@ -1,25 +1,26 @@
 # What is Fobnail Project?
 
-Fobnail Project provide resources to create axiomatically trustworthy
-[device][mccune] and simple user interface to attest platform state. For
-detailed project description please check [here](description).
+Fobnail Project provides resources to create axiomatically trustworthy
+[device][mccune] and simple user interface to attest platform state. For a
+detailed project description, please check [here](description).s
```

Comment:
@Stanislaw-bnk  the letter "s" at the end looks unnecessary.

Reply:
Good catch, the "s" has been removed https://github.com/fobnail/docs/pull/65/commits/34d6a7ec68f0f8811343937682c9db21c7f3c2b9